File: pickit/src/main/java/com/hbisoft/pickit/PickiTCallbacks.java
package com.hbisoft.pickit;

import java.util.ArrayList;

public interface PickiTCallbacks {
    void PickiTonUriReturned();
    void PickiTonStartListener();
    void PickiTonProgressUpdate(int progress);
    void PickiTonCompleteListener(String path, boolean wasDriveFile, boolean wasUnknownProvider, boolean wasSuccessful, String Reason);
    void PickiTonMultipleCompleteListener(ArrayList<String> paths, boolean wasSuccessful, String Reason);
}

File: pickit/src/main/java/com/hbisoft/pickit/PickiTCallbacks.kt
package com.hbisoft.pickit

interface PickiTCallbacks {
    fun PickiTonUriReturned()
    fun PickiTonStartListener()
    fun PickiTonProgressUpdate(progress: Int)
    fun PickiTonCompleteListener(path: String?, wasDriveFile: Boolean, wasUnknownProvider: Boolean, wasSuccessful: Boolean, Reason: String?)
}